### PR TITLE
add --ssh flag to bootstrap.py

### PR DESCRIPTION
**Pull request by jessfraz**
Allows for cloning private repos.

### CODE REVIEW

File: jenkins/bootstrap.py [hunk 1]
```diff
@@ -782,6 +787,10 @@ def parse_args(arguments=None):
     parser.add_argument(
         '--service-account',
         help='Activate and use path/to/service-account.json if set.')
+    parser.add_argument(
+        '--ssh',
+        nargs='?', const=1, default=0,
```
Comment (rmmh) [inline jenkins/bootstrap.py]: this should be `default=False, action='store_true'` for a boolean flag.

**Comment by rmmh** [comment]
This looks good other than the minor nit about argument parsing.

/lgtm

File: jenkins/bootstrap.py (hunk 1 above)
Comment (fejta) [inline jenkins/bootstrap.py]: store_true is less code, but for the person using this from a shell I kind of like being able to distinguish between "I don't care", "yes" and "no"

**Comment by fejta** [comment]
Jess if you won't have write access please let us know if you get this to a state where you want us to submit

**Comment by jessfraz** [comment]
cool will update!

**Comment by jessfraz** [comment]
fixed the flag to store_true

File: jenkins/bootstrap.py [hunk 2]
```diff
@@ -782,6 +787,10 @@ def parse_args(arguments=None):
     parser.add_argument(
         '--service-account',
         help='Activate and use path/to/service-account.json if set.')
+    parser.add_argument(
+        '--ssh',
+        default=False, action='store_true',
```
Comment (fejta) [inline jenkins/bootstrap.py]: store_true actually handles default=False for you

Comment (jessfraz) [inline jenkins/bootstrap.py]: will fix

**Pull request merged**